modules/rosa-sts-about-ocm-role.adoc

```diff
@@ -20,7 +20,7 @@ If you just created a Red Hat account that is not part of an existing organizati
 +
 * See "Understanding the {cluster-manager} role" in the Additional resources of this section for a list of the AWS permissions policies for the basic and admin `ocm-role` IAM resources.
 
-Using the `rosa` CLI, you can link your IAM resource when you create it.
+Using the {product-title} (ROSA) CLI (`rosa`), you can link your IAM resource when you create it.
```

Comment:
{product-title} should be in the first paragraph so that this can just be "ROSA CLI (`rosa`)".